dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -3710,6 +3710,24 @@ public void refresh(ContentType type) throws DotReindexStateException {
 
     }
 
+    @WrapInTransaction
+    @Override
+    public Contentlet saveContentOnVariant(final Contentlet contentlet, final String variantName,
+            final User user){
+
+        final String inode = contentlet.getInode();
+        final Contentlet checkedoutContentlet = Try.of(() -> checkout(inode, user, false))
```

Comment:
I think there is a typo here

Reply:
done https://github.com/dotCMS/core/pull/24450/commits/39c844389237f85a99ed82dc502d5018e20588a4#diff-fa1ceaa19618a6b2bbc30e24c6f930b4971f417db50babb748c2e2837ba9eb82L3718